analyzers/src/SonarAnalyzer.CSharp/Rules/TypeExaminationOnSystemType.cs

```diff
@@ -33,40 +33,33 @@ namespace SonarAnalyzer.Rules.CSharp
     [Rule(DiagnosticId)]
     public sealed class TypeExaminationOnSystemType : SonarDiagnosticAnalyzer
     {
-        internal const string DiagnosticId = "S3443";
+        private const string DiagnosticId = "S3443";
         private const string MessageFormat = "{0}";
-        internal const string MessageGetType = "Remove this use of 'GetType' on a 'System.Type'.";
-        internal const string MessageIsInstanceOfType = "Pass an argument that is not a 'System.Type' or consider using 'IsAssignableFrom'.";
-        internal const string MessageIsInstanceOfTypeWithGetType = "Consider removing the 'GetType' call, it's suspicious in an 'IsInstanceOfType' call.";
+        private const string MessageGetType = "Remove this use of 'GetType' on a 'System.Type'.";
+        private const string MessageIsInstanceOfType = "Pass an argument that is not a 'System.Type' or consider using 'IsAssignableFrom'.";
+        private const string MessageIsInstanceOfTypeWithGetType = "Consider removing the 'GetType' call, it's suspicious in an 'IsInstanceOfType' call.";
 
-        private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+        private static readonly DiagnosticDescriptor rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
```

Comment:
static readonly fields should begin with an upper-case letter.
```suggestion
        private static readonly DiagnosticDescriptor Rule = DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
```